docs/verbs/catkin_config.rst

```diff
@@ -189,12 +189,20 @@ To clear the blacklist, you can use the ``--no-blacklist`` option:
 Note that you can still build packages on the blacklist and whitelist by
 passing their names to ``catkin build`` explicitly.
 
+Appending or Removing List-Type Arguments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Several configuration options are actually *lists* of objects. For such
+options, you can use the ``--append-args`` and ``--remove-args`` options to
+append or remove elements from these lists, respectively.
```

Comment:
Should it (is it?) mentioned somewhere that the default behavior is to replace the list?